src/questionnaireitem_media_video.js

```diff
@@ -59,7 +59,23 @@ QuestionnaireItemMediaVideo.prototype._createMediaNode = function() {
 
     this.videoNode = document.createElement('video');
     this.videoNode.oncanplaythrough = this._onloaded.bind(this);
-    this.videoNode.src = this.url;
+
+    if (Array.isArray(this.url)) {
+        // If an array of video sources is provided
+        for (var i = 0; i < this.url.length; i++) {
+            videoSource = document.createElement("source");
+            videoSource.src = this.url[i];
+            this.videoNode.appendChild(videoSource);
+        }
+    } else {
+        videoSource = document.createElement("source");
+        videoSource.src = this.url;
+        this.videoNode.appendChild(videoSource);
+    }
+
+    pTag = document.createElement("p");
```

Comment:
Please move adding pTag to a separate commit and rename it to videoNotSupportMessage.
Then it's meaning is easier to understand.

Reply:
okay done.